### PR TITLE
added my profile sir

### PR DESCRIPTION
**Pull request by charan-web**
please accept my pull request

### CODE REVIEW

**Comment by eddiejaoude** [comment]
Please change from upper case to lower case in the pull request and code changes, we do not need to shout 😉

**Comment by charan-web** [comment]
i just changed it sir. please accept my request

File: data.json [hunk 1]
```diff
@@ -184,5 +184,13 @@
     "githubUsername": "NirmalSilwal",
     "imageUrl": "https://user-images.githubusercontent.com/31946761/93670031-aefc0d00-fab5-11ea-9ca9-cc4bfe73fa83.png",
     "issueNumber": 93
+   },
+    {
+    "name": "CHARAN",
```
Comment (eddiejaoude) [inline data.json:189]: Your name is still in capitals, please make lower case

**Comment by eddiejaoude** [comment]
Your name in the json file is still uppercase, please make it lowercase - I made an inline comment to remind you

File: data.json [hunk 2]
```diff
@@ -184,5 +184,13 @@
     "githubUsername": "NirmalSilwal",
     "imageUrl": "https://user-images.githubusercontent.com/31946761/93670031-aefc0d00-fab5-11ea-9ca9-cc4bfe73fa83.png",
     "issueNumber": 93
+   },
+    {
+    "name": "charan",
+    "githubUsername": "charan-web",
+    "imageUrl": "https://user-images.githubusercontent.com/56212945/93706577-39d81880-fb45-11ea-9f6c-236275f1a0be.jpg",
+    "issueNumber": 95
```
Comment (eddiejaoude) [inline data.json:192]: Indentation is not correct, but that can be fixed in another PR

**Review by eddiejaoude — APPROVED** [review]

**Pull request merged**